consensus/state.go

```diff
@@ -1464,11 +1464,11 @@ func (cs *ConsensusState) recordMetrics(height int64, block *types.Block) {
 	missingValidators := 0
 	missingValidatorsPower := int64(0)
 	for i, val := range cs.Validators.Validators {
-		var vote *types.CommitSig
-		if i < len(block.LastCommit.Precommits) {
-			vote = block.LastCommit.Precommits[i]
+		if i >= len(block.LastCommit.Precommits) {
```

Comment:
I think the only case where len(Precommits) != len(Validators) is where height == 1 (due to first commit being empty)

Reply:
Will create another PR to address this.